pwiz.py

```diff
@@ -138,6 +141,8 @@ def get_option_parser():
     ao('-i', '--info', dest='info', action='store_true',
        help=('Add database information and other metadata to top of the '
              'generated file.'))
+    ao('-o', '--order-preserved', action='store_true',
```

Comment:
Hello Charles,

I�ll be happy to oblige on all three of your requested improvements. I�ll send you a new PR when it�s done. 

Thanks again for the INSERT � RETURNING version 2.5.0. It helps me a lot.

  Ed